tasks/go.py

```diff
@@ -428,3 +435,182 @@ def go_fix(ctx, fix=None):
             for osname in oslist:
                 tags = set(ALL_TAGS).union({osname, "ebpf_bindata"})
                 ctx.run(f"go fix{fixarg} -tags {','.join(tags)} ./...")
+
+
+def get_deps(ctx, path):
+    with ctx.cd(path):
+        # Might fail if no mod tidy
+        deps: list[str] = ctx.run("go list -deps ./...", hide=True, warn=True).stdout.strip().splitlines()
+        prefix = 'github.com/DataDog/datadog-agent/'
+        deps = [
+            dep.removeprefix(prefix)
+            for dep in deps
+            if dep.startswith(prefix) and dep != f'github.com/DataDog/datadog-agent/{path}'
+        ]
+
+        return deps
+
+
+def add_replaces(ctx, path, replaces: list[str]):
+    repo_path = os.path.abspath('.')
+    with ctx.cd(path):
+        for repo_local_path in replaces:
+            if repo_local_path != path:
+                # Example for pkg/util/log with path=pkg/util/cachedfetch
+                # - repo_local_path: pkg/util/log
+                # - online_path: github.com/DataDog/datadog-agent/pkg/util/log
+                # - module_local_path: ../../../pkg/util/log
+                level = os.path.abspath(path).count('/') - repo_path.count('/')
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Since on Windows the `os.path.abspath('.')` might return a string with `\`s, wouldn't the `repo_path.count('/')` be 0 always?

Reply:
You might want to use `posixpath` since it has the same interface but for UNIX style.